src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -277,6 +285,17 @@ def query_top_tags(
 
     with sentry_sdk.start_span(op="discover.discover", description="facets.top_tags"):
 
+        if not orderby:
+            orderby = "-count"
+
+        if len(orderby) >= 1:
```

Comment:
I think this will break
since `len("-count") >= 1` -> `orderby="-"`
```suggestion
        if isinstance(orderby, list) >= 1:
```

Reply:
What's the intent behind only allowing one orderby for top_tags?

Reply:
Just wanted to limit it for now since I don't plan on allowing passing more then one order for the table at the moment.

Reply:
Is it worth just keeping the additional orderbys for now? I don't think there's any major drawback, and saves a future PR to allow >1 orderby